data_filter_elasticsearch/internal/api/api.go

```diff
@@ -59,12 +61,27 @@ func New(esClient *elastic.Client, index string) *ServerAPI {
 
 // Run the server.
 func (api *ServerAPI) Run(ctx context.Context) error {
+	fmt.Println("Loading OPA SDK....")
```

Comment:
Since we're using the `log` package above, perhaps use that here and `log.Debug` or something?